### PR TITLE
The first section (hero-section) of the main page has been created. 

### PR DESCRIPTION
**Pull request by min-in-zon**
The search bar was also created as a separate reusable component.
SCSS variables were created for colors and fonts to make styling easier.

### CODE REVIEW

File: src/components/common/SearchPannel/SearchPannel.jsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import './searchPannel.scss';
+
+const SearchPannel = () => {
+  return (
+    <form className="search-container" action="/search" method="GET">
```
Comment (adadychanka) [inline src/components/common/SearchPannel/SearchPannel.jsx:5]: Discussion about wrapping search input in a form element

https://stackoverflow.com/a/51691457/10297026

File: src/components/common/SearchPannel/SearchPannel.jsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+      <button type="submit" className="search-button">
```
Comment (adadychanka) [inline src/components/common/SearchPannel/SearchPannel.jsx:6]: I don't think a search input works that way by clicking on the magnifier icon

Check this example
https://mui.com/material-ui/react-autocomplete/#search-input

Comment (adadychanka) [reply src/components/common/SearchPannel/SearchPannel.jsx:6]: You might see the similar experience in LinkedIn

image

File: src/components/common/SearchPannel/SearchPannel.jsx [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+        <img src="../../../src/assets/img/search-icon.png" alt="Search icon" />
+      </button>
+      <input
+        type="text"
```
Comment (adadychanka) [inline src/components/common/SearchPannel/SearchPannel.jsx:10]: It's better to use existing type `search`

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/search

File: src/components/common/SearchPannel/SearchPannel.jsx (hunk 2 above)
Comment (adadychanka) [inline src/components/common/SearchPannel/SearchPannel.jsx:6]: Usually, if button contains only an icon without a label, it should have `aria-label` attribute describing the action that button do

File: src/components/common/SearchPannel/searchPannel.scss [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+        align-items: center;
+        justify-content: center;
+        padding: 0;
+        border: none;
+        background-color: transparent;
+        width: 1rem;
+        height: 1.06rem;
+        
+        &:hover {
+            cursor: pointer;
```
Comment (adadychanka) [inline src/components/common/SearchPannel/searchPannel.scss:25]: Button element has `cursor: pointer` by default

File: src/components/common/css/logoButton.min.css
Comment (adadychanka) [inline src/components/common/css/logoButton.min.css:1]: It has to be deleted

File: src/components/common/css/logoButton.min.css.map
Comment (adadychanka) [inline src/components/common/css/logoButton.min.css.map:1]: It has to be deleted

File: src/components/common/css/navBar.min.css
Comment (adadychanka) [inline src/components/common/css/navBar.min.css:1]: It has to be deleted

File: src/components/common/css/navBar.min.css.map
Comment (adadychanka) [inline src/components/common/css/navBar.min.css.map:1]: It has to be deleted

File: src/components/common/css/searchPannel.min.css
Comment (adadychanka) [inline src/components/common/css/searchPannel.min.css:1]: It has to be deleted

File: src/components/common/css/searchPannel.min.css.map
Comment (adadychanka) [inline src/components/common/css/searchPannel.min.css.map:1]: It has to be deleted

File: src/pages/css/heroSection.min.css
Comment (adadychanka) [inline src/pages/css/heroSection.min.css:1]: It has to be deleted

File: src/pages/css/heroSection.min.css.map
Comment (adadychanka) [inline src/pages/css/heroSection.min.css.map:1]: It has to be deleted

**Review by adadychanka — CHANGES_REQUESTED** [review]
It seems you ran sass and compiled your scss files.

These result files should be only in a prod build, not in development
```
.min.css
.min.map

```

File: src/components/common/SearchPannel/searchPannel.scss (hunk 4 above)
Comment (min-in-zon) [reply src/components/common/SearchPannel/searchPannel.scss:25]: Not at all.

**Review by adadychanka — APPROVED** [review]

**Pull request merged**